### DISCUSSION TITLE
Incremental builds coming soon

### DISCUSSION THREAD

**Discussion by sotteson1** [microsoft/win32metadata#669] [Ideas] [answered]
I've done a bunch of work that will bring incremental builds for both the main windows.win32.winmd and external projects that use the the generator. Because I wanted windows.win32.winmd to use the same build system as external projects, I had to move some things around a bit. Also, this will bring multi-arch scraping to external projects. As part of the coming change, I've tried to update the docs to reflect the new changes, but if I've missed something, let me know.

**Comment by mikebattista** [comment, marked as answer]
Thanks. Trying it now and will update the docs based on my experience.
